configuration.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
should this be in gitignore?

Reply:
I think it's better to have a default one already there, but renaming it is a good idea. Done in e38e46812c6005656c62152a213db00f960750ed.